apps/service_acquire/lib/service_acquire/presto.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Acquire.Presto do
```

Comment:
Should we use a more generic term for the name of the behaviour, Acquire.Query or something like that so behaviour is not tied to presto

Reply:
`Acquire.DataStore`

Reply:
Or if you prefer later with more options: `Acquire.DB.Relational` vs `Acquire.DB.Graph`

Reply:
I am good with any of those.